src/industry_cmd.cpp

```diff
@@ -1990,7 +1993,7 @@ CommandCost CmdBuildIndustry(TileIndex tile, DoCommandFlag flags, uint32 p1, uin
 	randomizer.SetSeed(p2);
 	uint16 random_initial_bits = GB(p2, 0, 16);
 	uint32 random_var8f = randomizer.Next();
-	int num_layouts = indspec->num_table;
+	int num_layouts = (int)indspec->layouts.size();
```

Comment:
feel like this variable should be a uint/size_t

Reply:
I should give it all another pass for consistent integer widths and signedness, yes.